consensus/src/experimental/execution_phase.rs

```diff
@@ -18,54 +18,107 @@ use std::sync::Arc;
 /// ExecutionPhase sends the ordered blocks to the commit phase.
 ///
 
-pub type ExecutionChannelType = (
-    Vec<Block>,
-    LedgerInfoWithSignatures,
-    StateComputerCommitCallBackType,
+pub type ResetEventContentType = ();
+pub fn reset_event_content_new() -> ResetEventContentType {}
```

Comment:
don't need it to be a function? can just use ResetEventContentType

Reply:
Just in case we might need to put something in the reset ack.